recipes/tinker/meta.yaml

```diff
@@ -0,0 +1,52 @@
+{% set name = "tinker" %}
+{% set version = "8.6.1" %}
+{% set sha256 = "395556fd1a4c4907ab3f3207487db536693df883db71986a896ba1415f15fb6b" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://dasher.wustl.edu/tinker/downloads/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  skip: true  # [win]
+  script:
+    - cd source
+    - cp ../make/Makefile .
+    - make TINKERDIR=$PREFIX RANLIB=ranlib LIBDIR="-L. -L$PREFIX/lib/linux" F77=$GFORTRAN FFTWDIR="$PREFIX"  # [linux]
+    - make TINKERDIR=$PREFIX F77=$GFORTRAN FFTWDIR="$PREFIX" LINKFLAGS='$(OPTFLAGS) -static-libgcc -headerpad_max_install_names'  # [osx]
+    - make install TINKERDIR=$PREFIX
+
+requirements:
+  build:
+    - {{ compiler("fortran") }}
+    - make
+  host:
+    - fftw
+  run:
+    - openmp
+    - fftw
+
+test:
+  source_files:
+    - test
+  commands:
+    - cd test
+    - cp anion.dyn0 anion.dyn
+    - dynamic anion 1000 2.0 0.1 2 300.0
+    - alchemy < anion.dat
+
+about:
+  home: https://dasher.wustl.edu/tinker/
+  license: custom
+  license_file: LICENSE.pdf
```

Comment:
[From the license:](https://github.com/TinkerTools/Tinker/blob/release/LICENSE.pdf)

> The Licensee may not sublicense the use of the Software, or make it available to third parties, without the prior written consent of at least one TCDC-Member.

I think written permission is needed from a TCDC member stating that conda-forge and numFOCUS are allowed distribute this software.